code/dynamic_programming/src/coin_change/coinchange.cpp

```diff
@@ -3,23 +3,22 @@
 
 #include<iostream>
 #include<vector>
-using namespace std;
 
-const int MAX=100;
+const int MAX = 100;
 
-int coinWays(int amt,vector<int>& coins){
+int coinWays(int amt,std::vector<int>& coins){
```

Comment:
Add space after comma.
Add newline between close parenthesis and open brace.
And also remove blank line below.